ci-build/docker-run.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+shopt -s extglob
+
+HTML_SOURCE=$(realpath "$1")
+HTML_OUTPUT=$(realpath "$2")
+
+docker run --rm --interactive --tty \
```

Comment:
Should `--interactive --tty` be here, or was that for testing?

Reply:
Interesting. I'm a bit surprised this worked on Travis; GitHub actions doesn't like it when I use `--tty`. So yeah, let's delete these. Thanks for catching.